src/dataExplorer/components/FluxQueryBuilder.tsx

```diff
@@ -60,10 +51,24 @@ const FluxQueryBuilder: FC = () => {
             data-testid="flux-query-builder--menu"
           >
             <Button
-              onClick={clear}
+              onClick={() => setIsPromptVisible(true)}
```

Comment:
Ah...I believe we need to still run `clear()`, because that also resets the state used for schema composition.